api/src/main/java/org/cardanofoundation/rosetta/common/interceptor/CustomResponseBodyAdviceAdapter.java

```diff
@@ -14,12 +15,11 @@
 import org.cardanofoundation.rosetta.common.services.LoggingService;
 
 @ControllerAdvice
+@RequiredArgsConstructor
 public class CustomResponseBodyAdviceAdapter implements ResponseBodyAdvice<Object> {
 
-    @Autowired
-    LoggingService loggingService;
+    final LoggingService loggingService;
 
-    @Override
```

Comment:
Are you sure that it's not overriding (implementing) a parent method anymore? 🤔

Reply:
Good point